src/plugins/Migrations/controllers/commands/migrateGuilds.js

```diff
@@ -0,0 +1,129 @@
+const db = require("quick.db");
+const dbh = require("../../../.common/structures/DataHandling/DatabaseHandler.js");
+const { Guild, Channel } = require("./../../../../../api/models");
+const migrationsHandler = require("./../../structures/MigrationHandler.js");
+
+exports.help = {
+  name: "migrateGuilds",
+  description: "Migrates all the data for the guilds",
+  usage: "migrateGuilds",
+  example: "migrateGuilds",
+};
+
+exports.conf = {
+  aliases: [],
+  cooldown: 5,
+  permissions: ["ADMINISTRATOR"],
+};
+
+exports.run = async (client, message, args) => {
+  const guilds = client.guilds.cache;
+  for (const guild of guilds) {
+    const guildId = message.guild.id;
+    const guildInDb = await Guild.findOne({ where: { id: guildId } });
+
+    if (guildInDb !== null) {
+      message.channel.send(`${guild[1].name} already has been created`);
+      continue; //Skips to the next guild;
+    }
+
+    /*For things that can be copy-pasted, with no modifications being made to the data,
+        we will use the system below
+      */
+    let keys = [
+      { oldKey: `prefix.${guildId}`, newKey: "prefix" },
+      { oldKey: `${guildId}.newUserRoles`, newKey: "defaultMemberRoles" },
+      { oldKey: `moderation-server.${guildId}`, newKey: "moderationServer" },
+    ];
+
+    const guildObject = migrationsHandler.copyData(keys);
+
+    guildObject.id = guildId;
+
+    const opRoles = [];
+    const modRole = db.get(`moderator.${guildId}`);
```

Comment:
We are appending `guildId` in many places, we can create a function to do that.

Reply:
How would that look? The issue is that in some cases we are appending `guildId` in a different order, sometimes before and sometimes after the word.